README.md

```diff
@@ -20,6 +20,15 @@ Well, with `what` all you have to do is ask `what "5f4dcc3b5aa765d61d8327deb882c
 
 `what`'s job is to **identify _what_ something is.** Whether it be a file or text! Or even the hex of a file! What about text _within_ files? We have that too! `what` is recursive, it will identify **everything** in text and more! 
 
+
+# :question: Installation 
+#### Windows
+ `pip install pyWhat`
```

Comment:
This depends on whether python 2 or 3 is installed, `pip3` will always equal the python3 version 😄